charts/mod-aes/values.yaml

```diff
@@ -58,6 +58,18 @@ ingress:
 
 args: []
 
+eureka:
```

Comment:
Here in values it no need to put all vars

it could be better to have full config in example chart only

eureka: 
  enabled: false
  sidecarContainer: {}